x/staging/subspaces/client/cli/keys.go

```diff
@@ -0,0 +1,6 @@
+package cli
+
+// Subspaces flag
+const (
+	FlagNewName = "new-name"
```

Comment:
I think we should have simpler flag name such as `--name` instead of `--new-name`. Also, is it necessary to put the flags here? Can't we declare them in the same place where the command is used?

Reply:
we declared them like this inside the post's cli commands, I was following the same pattern